### DISCUSSION TITLE
X bar representing the mean...

### DISCUSSION THREAD

**Discussion by dsantiago** [typst/typst#1178] [General]
Hi all,

I did something like this to represent the X bar:
<img width="80" alt="image" src="https://github.com/typst/typst/assets/3484029/ba048a9c-79a4-4392-9685-a7282cbb38f9">

```rust
#overline()[_X_]
```

Isn't a way to do this inside equations with a proper math syntax?

**Comment by Enivex** [comment]
`$macron(x)$` or `$overline(x)$`

**Comment by dsantiago** [comment]
So there is a specific overline function for equations? I did not found it int the docs, would you please point me if it's there in the documentation?

**Comment by Enivex** [comment]
https://typst.app/docs/reference/math/underover/

**Comment by dsantiago** [comment]
Found it right now as was digging more... So, there are a text overline and math overline... 

Thanks anyway for the help...
